### ISSUE TITLE
Issue - Grammatical issues and wordy sentences

### ISSUE THREAD

**Issue by Marcus5408** [Avdan-OS/Website#199]
**Issue:**
Many sentences on the Features page have bad grammar and/or are hard to understand.

**To Reproduce:**
Steps to reproduce:
1. Go to avdanos.com
2. Click on "Features"
3. Scroll down
4. See errors

**Expectations:**
The text on the page has correct grammar and is easy to understand.

**System Info:**
- Device: iPhone 7 Plus
- OS: iOS 15.5
- Browser: Default, Safari
- Version: 15.5

**Comment by AZProductions** [comment]
Yes, thank you. The grammar should be improved (but with that, all of the translations need to update which is alot of work). I'll mark this as a grammar/translations issue.

**Comment by Marcus5408** [comment]
Update: It's been a while since I last checked in with this project. Issue still exists on newer hardware.

Updated Hardware:
- Device: iPhone 14 Plus
- OS: iOS 17.1 Developer Beta
- Browser: Stock Browser, Safari
- Version: 17.1

![image](https://github.com/Avdan-OS/Website/assets/46655509/a779ba10-9971-4458-a1e5-6111f56e1e53)
